docs/Homebrew-homebrew-core-Merge-Checklist.md

```diff
@@ -1,22 +1,128 @@
-# Homebrew/homebrew-core Merge Checklist
+# Core Maintainer Guide
 
-The following checklist is intended to help maintainers decide on
-whether to merge, request changes or close a PR. It also brings more
-transparency for contributors in addition to
-[Acceptable Formulae](Acceptable-Formulae.md) requirements.
+## Quick merge checklist
+
+A detailed checklist can be found [below](#detailed-merge-checklist). This is all that really matters:
+
+- Ensure the name seems reasonable.
```

Comment:
```suggestion
- Ensure the name seems reasonable. Formulae that are also packaged by other package managers (e.g. Debian, Ubuntu) should be named consistently (subject to minor differences due to Homebrew formula naming conventions).
```
Actually, maybe this needs to go into the `Naming` section below (with a link here to down there?).